src/ufbcnt.f

```diff
@@ -33,6 +24,15 @@
 C> - The value returned for KMSG does <b>not</b> include any messages
 C> which contain DX BUFR tables information.
 C>
+C> @param[in] LUNIT -- integer: Fortran logical unit number for BUFR file
+C> @param[out] KMSG -- integer: Ordinal number of current message,
+C>                     counting from the beginning of the BUFR file, but
+C>                     not counting any messages which contain DX BUFR
+C>                     tables information
+C> @param[out] KSUB -- integer: Ordinal number of current data subset
+C>                     within (KMSG)th message, counting from the
+C>                     beginning of the message
+C>
 C> @author J. Woollen @date 1994-01-06
```

Comment:
Just curious, why are you moving the parameter descriptions in all of these routines to be _after_ the remarks and discussion?  The remarks and discussion notes refer to some of the parameters listed in the parameter descriptions, so it seems more sensible to put the notes _after_ the descriptions; otherwise, they're talking about concepts that haven't even been introduced yet!

Reply:
That's the way it is done in all the NCEPLIBS and I'm trying to maintain a similar look and feel everywhere. Description, parameters, return codes, author is how we do it everywhere else.